app/scripts/controllers/journalUpload.js

```diff
@@ -0,0 +1,241 @@
+import registrationsPaidPopoverTemplate from 'views/components/registrationsPaidPopover.html';
+import paymentsModalTemplate from 'views/modals/paymentsModal.html';
+import journalUploadReviewModalTemplate from 'views/modals/journalUploadReview.html';
+
+angular
+  .module('confRegistrationWebApp')
+  .controller('journalUploadCtrl', function(
+    $rootScope,
+    $scope,
+    $http,
+    $window,
+    $cookies,
+    $uibModal,
+    registrations,
+    reports,
+    journalUploadService,
+    conference,
+    envService,
+    permissions,
+    modalMessage,
+  ) {
+    $rootScope.globalPage = {
+      type: 'admin',
+      mainClass: 'journal-upload',
+      bodyClass: '',
+      confId: conference.id,
+      footer: true,
+    };
+    $scope.paidPopoverTemplateUrl = registrationsPaidPopoverTemplate;
+    $scope.accountTransfers = journalUploadService.getAccountTransferData(
+      registrations,
+    );
+    $scope.accountTransfersWithErrors = journalUploadService.getAccountTransferDataWithErrors(
+      registrations,
+    );
+    $scope.registrations = registrations;
+    $scope.reports = reports;
+    $scope.currentReportId = null;
+    $scope.conference = conference;
+    $scope.accountTransfersToInclude = [];
+    $scope.queryParameters = {
+      page: 1,
+      limit: 20,
+      orderBy: 'last_name',
+      order: 'ASC',
+      filter: '',
+      filterAccountTransferErrors: 'yes',
+      filterAccountTransfersByExpenseType: '',
+      filterAccountTransfersByPaymentType: '',
+      filterPayment: '',
+      filterRegType: '',
+      includeAccountTransfers: true,
+      includeCheckedin: 'yes',
+      includeWithdrawn: 'yes',
+      includeIncomplete: 'yes',
+    };
+    $scope.meta = {
+      totalPages: 0,
+    };
+    $scope.apiUrl = envService.read('apiUrl');
+    $scope.authToken = $cookies.get('crsToken');
+
+    $scope.$watch(
+      'queryParameters',
+      (newParameters, oldParameters) => {
+        //limit changed, reset page
+        if (
+          newParameters.page > 1 &&
+          newParameters.page === oldParameters.page
+        ) {
```

Comment:
I based this on how it works for the existing payments page: https://github.com/CruGlobal/conf-registration-web/blob/db6dd278a4d632890f15f2bd00c6556874e6020b/app/scripts/controllers/paymentReport.js#L41-L44
Technically if they change the limit, that will cause a refresh of the data.  But I think what's of most concern is resetting them back to the first page if they currently are not already on it which is why ```page``` was chosen in favor ```limit``` but I think it could work either way really.